0000-template.md

```diff
@@ -1,23 +1,31 @@
 ---
-Stage: Accepted
-Start Date: 
-Release Date: Unreleased
-Release Versions:
+stage: accepted
+start-date: 
+release-date: 
+release-versions:
   ember-source: vX.Y.Z
   ember-data: vX.Y.Z
-Relevant Team(s): 
-RFC PR: 
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+proposal-pr:
+tracking-link:
```

Comment:
Why add back tracking link? In theory it should be unnecessary because the stages (especially 'Ready for Release') are the tracking.

Reply:
I believe the idea here is when an RFC is merged this would point to one of the following:
- Quest Issue
- PR
- Github Project

In any of these cases the RFCs app would be able to use the Github API to check completion progress.

Reply:
Ah, my understanding of the Stages RFC was the goal to replace the `Tracking link` with tracking of the progress done on the PR to move the RFC from `Accepted` to `Ready for Release`

Reply:
If we want to keep, I think we should make it an array field so there can be many links

Reply:
I'm somewhat flexible on this and open to whatever will ultimately work best.

Reply:
So this is a bit of a crossover between two efforts that are going on here. There was the RFC Stages RFC that talks about the PRs to move from one stage to the next but there is also the Ember Learning Initiative that I proposed (way back in 2019) to communicate the progress of RFCs using an "app". Imagine the next phase of https://rfcs.emberjs.com 

The idea of having a tracking link here is to have something that is keeping track of the next steps, as @wagenet mentions I have in mind that it could be an Issue a PR or a Github project depending on the preference of the person/team that is implementing the RFC. With a single tracking link we are able to use this information and show it on the app as some sort of progress-bar percentage (or something) which is one of the key goals of what I'm trying to do with my Initiative.

I hope this makes sense, let me know if you have any more questions and I can try to explain my point of view a bit better 👍

Reply:
We previously had tracking links and used the RFC Tracking Repo. Stages is supposed to be a replacement of sort that would require the teams to update it to move the RFC forward -- the various links and status would be collected in the PR to the `Ready for Release` stage. https://github.com/emberjs/rfcs/blob/master/text/0617-rfc-stages.md#changes-to-rfc-meta

Reply:
I have updated this based on the Discord discussion. Now we have a `prs` object that will have keys that will each point to a PR for advancement to that stage. I have added a `project-link` as a compromise to allow projects to track progress in the website 👍